README.md

```diff
@@ -20,8 +25,11 @@ npm v8.3
 
 Clone the repository: 
 
-`git clone https://github.com/gdumani/math-magician.git`
-`cd math-magician`
+`git clone https://github.com/gdumani/my-mathmagician.git`
+
+`cd my-mathmagician`
+
+`git checkout feat/site`
 
 Load required npm software
 
```

Comment:
- [ ] Please add the new [Microverse README Template](https://github.com/microverseinc/readme-template) to your project.